src/theme.js

```diff
@@ -45,80 +53,152 @@ export default (overrides = {}) => {
 
   const typography = {
     fontSize: 12,
+    bodyFontFamily: 'Avenir',
+    headerFontFamily: 'Tiempos',
   };
 
   const buttonVariants = {
     primary: {
       backgroundColor: colors.primary,
-      fontColor: 'white',
+      fontColor: colors.white,
+      hover: {
+        backgroundColor: colors.primaryLight,
+      },
+      active: {
+        backgroundColor: colors.primaryDark,
+      },
+      disabled: {
+        backgroundColor: colors.primaryLightest,
+      },
     },
-    success: {
-      backgroundColor: colors.green,
-      fontColor: 'white',
+    secondary: {
+      backgroundColor: colors.white,
+      fontColor: colors.primary,
+      borderColor: colors.primary,
+      textColor: colors.primary,
```

Comment:
font color and text color? can these be consolidated?

Reply:
Well, I think we could probably come up with a better variable name. The purpose of `textColor` is an override for when the button is a Text Button. For the case of the `primary` button, `fontColor` is white. that won't work for a Text Button. 

I thought about using a function to determine this, but in case someone actually wants a dark background and a white Text Button color, it would be best to leave it as a overridable variable.

Reply:
I agree with @cehsu here - fontColor and textColor will be too easy to conflate. I propose text buttons be their own variant, e.g. `greyText`, so we don't have to be too tricky with the `text` prop